patches/tModLoader/Terraria/GameContent/UI/States/UICharacterCreation.cs.patch

```diff
@@ -8,11 +8,14 @@
  using Terraria.UI;
  using Terraria.UI.Gamepad;
  
-@@ -1053,6 +_,7 @@
+@@ -1053,6 +_,10 @@
  
  			_player.savedPerPlayerFieldsThatArentInThePlayerClass = new Player.SavedPlayerDataWithAnnoyingRules();
  			CreativePowerManager.Instance.ResetDataForNewPlayer(_player);
-+			PlayerHooks.SetStartInventory(_player, PlayerHooks.SetupStartInventory(_player));
++
++			var inventoryAdditions = PlayerHooks.AddStartingItems(_player);
++			PlayerHooks.ModifyStartingInventory(_player, inventoryAdditions);
++			PlayerHooks.SetStartInventory(_player, inventoryAdditions.Values.SelectMany(i => i).ToList());
```

Comment:
Can't you just do `inventoryAdditions.Values.ToList()`?
Oh wait, I guess SelectMany is flattening it...

Reply:
Yes, it flattens the list